rust/worker/src/execution/operator.rs

```diff
@@ -54,13 +56,18 @@ where
         let res = self.reply_channel.send(output).await;
         // TODO: if this errors, it means the caller was dropped
     }
+
+    fn getTracingContext(&self) -> Option<tracing::Id> {
+        self.tracing_context.clone()
+    }
 }
 
 /// Wrap an operator and its input into a task message.
 pub(super) fn wrap<Input, Output, Error>(
```

Comment:
It it possible to make tracing context a part of wrap() in `system/`'s definition of a message instead of here? That way all users of system get tracing not just users of the operator/executor